pypreprocess/nipype_preproc_spm_utils.py

```diff
@@ -114,10 +134,10 @@ def _configure_backends(spm_dir=None, matlab_exec=None, spm_mcr=None,
 
 
 def _do_subject_slice_timing(subject_data, TR, TA=None, spm_dir=None,
-                             matlab_exec=None, spm_mcr=None, ref_slice=0,
+                             matlab_exec=None, spm_mcr=None, ref_slice=1,
```

Comment:
Can you explain why we need ref_slice=1 ?

Reply:
I decided to set this parameter to 1 in order to have the same default values in all functions: check for instance the do_subject_preproc function defaults.
On to of that the test on line 148 (master) / 168 (PR) is not valid if initialized with the default ref_slice value: assert 1 <= ref_slice <= nslices, ref_slice